api/Models/Civil/Detail/JudicialBinderDocumentDto.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Scv.Api.Models.Civil.Detail;
+
+public class JudicialBinderDocumentDto
+{
+    public string CivilDocumentId { get; set; }
+    public int Order { get; set; }
+    public CivilDocument Document { get; set; }
+}
```

Comment:
This should be moved out of Civil Detail and turned into a generic BinderDocumentDto.